server/bin/org_migrator.py

```diff
@@ -827,34 +843,56 @@ def main():
 
     if options.debug:
         log.setLevel(logging.DEBUG)
+        cp.set_log_level(logging.DEBUG)
 
     db = None
 
     try:
         db = cp.get_db_connector(options.dbtype, options.host, options.port, options.username, options.password, options.db)
-        log.info('Using backend: %s', db.backend())
-
-        org_id = resolve_org(db, args[0])
-        log.info('Resolved org "%s" to org ID: %s', args[0], org_id)
+        log.info('Using database: %s @ %s', db.backend(), options.host)
 
         if options.act_import:
-            importer = OrgImporter(db, options.file, org_id)
+            # Use the provided argument if the --file is still default
+            if options.file == 'export.zip' and args[0]:
```

Comment:
If I run it like this (assuming I have an export.zip in my current dir) `org_migrator.py --import`, this throws `IndexError: list index out of range`. Need to check the length of args first.

Reply:
Fixed

Reply:
I get this now:
```
[vagrant@candlepin vagrant]$ ./server/bin/org_migrator.py --import
2019-03-20 15:12:23,688 INFO    org_migrator     Using database: PostgreSQL @ localhost
Traceback (most recent call last):
  File "./server/bin/org_migrator.py", line 903, in <module>
    main()
  File "./server/bin/org_migrator.py", line 860, in main
    if not os.access(options.file, os.R_OK) or not os.isfile(options.file):
AttributeError: 'module' object has no attribute 'isfile'
```
(this is with python2.7)
I think you might need to do `os.path.isfile` instead